src/cdiserrors.py

```diff
@@ -116,17 +122,34 @@ def __init__(self, message, e=None):
         message = "{}: {}".format(message, e) if e else message
         super(SchemaError, self).__init__(message)
 
+
 class UnhealthyCheck(APIError):
     def __init__(self, message):
         self.message = str(message)
         self.code = 500
 
-def make_json_error(ex):
-    response = jsonify(message=str(ex))
-    response.status_code = (
-        ex.code if isinstance(ex, HTTPException) else 500)
-    return response
 
-def setup_default_handlers(app):
-    for code in default_exceptions:
-        app.register_error_handler(code, make_json_error)
+try:
+    from flask import jsonify
+    from werkzeug.exceptions import default_exceptions
+    from werkzeug.exceptions import HTTPException
+
+    def make_json_error(ex):
+        response = jsonify(message=str(ex))
+        response.status_code = ex.code if isinstance(ex, HTTPException) else 500
+        return response
+
+    def setup_default_handlers(app):
+        for code in default_exceptions:
+            app.register_error_handler(code, make_json_error)
+
+
+except ImportError:
+
+    # noinspection PyUnusedLocal
```

Comment:
personal opinion: I would prefer to keep these linter comments out of the codebase unless we are officially adopting a specific linter across the board, could get messy